tools/hicup/hicup_truncater.xml

```diff
@@ -0,0 +1,59 @@
+<tool id="hicup_truncater" name="Truncater" version="0.1.0">
+    <macros>
+        <import>hicup_macros.xml</import>
+    </macros>
+    <expand macro="requirements_hicup" />
+    <expand macro="stdio" />
+    <command><![CDATA[
+        export FONTCONFIG_FILE=\$CONDA_PREFIX/etc/fonts/fonts.conf &&
+        export FONTCONFIG_PATH=\$CONDA_PREFIX/etc/fonts/ &&
+        hicup_truncater --re1 '$re1' $advanced_options.nofill 
+        #if $advanced_options.sequences:
+            --sequences '$advanced_options.sequences'
+        #end if
+        '$input_first_sequence' '$input_second_sequence'
+
+        && mv hicup_truncater_summary* hicup_truncater_summary.txt
+        && trunc_result_1=\$(echo '$input_first_sequence' | rev | cut -d'/' -f1 | rev)
+      
+        && mv \$trunc_result_1.trunc.fastq dataset1.trunc.fastq
+        && trunc_result_2=\$(echo '$input_second_sequence' | rev | cut -d'/' -f1 | rev)
+        && mv \$trunc_result_2.trunc.fastq dataset2.trunc.fastq
+        && mv \$trunc_result_1.truncation_barchart.svg dataset1.truncation_barchart.svg
+        && mv \$trunc_result_2.truncation_barchart.svg dataset2.truncation_barchart.svg
+        
+    ]]></command>
+    <inputs>
+        <expand macro="input_files" />
+        <expand macro="re1" />
+        <section name="advanced_options" title="Advanced options">   
+            <expand macro="no_fill" />     
+            <param argument="--sequences" type="text" value="" label="Ligation sequence" help="Instead of specifying a restriction enzyme recognition sequence, specify the ligation sequences directly"/>
```

Comment:
As far as I understand the documentation they expect a text string which is the ligation sequence. It is not that well documented :(
http://www.bioinformatics.babraham.ac.uk/projects/hicup/scripts_description/#Truncater